temboardui/model/__init__.py

```diff
@@ -68,3 +73,35 @@ def worker_engine(dbconf):
     """
     dsn = 'postgresql://{user}:{password}@:{port}/{dbname}?host={host}'
     return create_engine(dsn.format(**dbconf))
+
+
+def check_schema(config):
```

Comment:
I wonder if this command could be implemented using `alembic.command.current()` (and possibly other built-in introspection commands).

Reply:
Il faudrait mocker la *fonction dans la fonction* `display_version`. Je me suis basé sur cette fonction.

https://github.com/sqlalchemy/alembic/blob/master/alembic/command.py#L503-L515

Reply:
Ok, dans ce cas peut-être l'indiquer en commentaire dans la fonction (pour que l'on sache d'où ça vient).

Reply:
Fait.